--- GERENCIADOR INICIADO COM SUCESSO ---
 Arquivo 'documento_teste_erros (1)_gDylcya.docx' copiado para a pasta 'C:\Users\Danilo\Downloads\dossel_project\media\entrada'.

 Documento a processar: ['documento_teste_erros (1)_gDylcya.docx']
 Modo de revisão: completa
 E-mail destino: n.danilo@dosselambiental.com.br
 Progresso atualizado para: 5% (ID da Revisão: 35)

 Iniciando processamento paralelo de 'Scripts\Mapeador_Completo.py' para 1 documento(s).

 Executando: C:\Users\Danilo\Downloads\dossel_project\venv\scripts\python.exe -u Scripts\Mapeador_Completo.py documento_teste_erros (1)_gDylcya.docx --id_revisao 35
 Cancelamento detectado para a revisão 35.
 Cancelamento solicitado; encerrando subprocesso …
 Houve falhas durante o processamento paralelo de 'Scripts\Mapeador_Completo.py'.
 Falhas durante o processamento. O processo pode ter sido cancelado.
 Progresso atualizado para: 100% (ID da Revisão: 35)
 Gerenciador finalizou a execução.
